__tests__/index.test.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
```

Comment:
we are not, to be honest, is a new dir where bit by bit I am going to move our tests to. it is the default dir name that jest uses in order to look for tests.